boards/v2/intel/intel_ish/Kconfig.defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if TEST
+config TEST_EXTRA_STACK_SIZE
```

Comment:
minor nit, the construct `if` construct is equivalent to apply `if` on all entries.

So when there is only one entry, then you might just write:
```
config TEST_EXTRA_STACK_SIZE
	int
	default 1024
	depends on TEST
```
(`depends on` is also equivalent to apply `if` on all parts)
Ref: https://www.kernel.org/doc/html/latest/kbuild/kconfig-language.html#